pms/tests/test_pms_folio_invoice.py

```diff
@@ -95,7 +92,6 @@ def create_common_scenario(self):
 
     def test_invoice_full_folio(self):
         # ARRANGE
```

Comment:
a short descriptive text of the task of the test can help a lot to understand what is being tested